### DISCUSSION TITLE
How to read and write values of type bool using snap7 module?

### DISCUSSION THREAD

**Discussion by TianSong1991** [gijzelaerr/python-snap7#469] [Q&A] [answered]
In Python, snap7 does not have a direct API to read and write bool type values. How to implement this? Snap7 needs to convert bool type values to read. Is there a direct solution to solve it? Use another method. How should snap7 write a bool type value, thank you.

**Comment by nikteliy** [comment, marked as answer]
Hi, @TianSong1991 
Check out  [this](https://github.com/gijzelaerr/python-snap7/blob/master/example/boolean.py) example. Is this what you are looking for?

**Reply by TianSong1991** [reply]
Thank you for your replying. I miss that example.
